client/components/Byline/Byline.tsx

```diff
@@ -72,6 +119,23 @@ const Byline = (props: PrivateBylineProps) => {
 		if (typeof contributors === 'string') {
 			return contributors;
 		}
+		if (bylinePrefix === BylineOption.Az || BylineOption.Za) {
+			// ReviewNote: Do we need to hande the case where a contributor is a string?
+			contributors.sort((last: any, next: any) => {
```

Comment:
We should avoid sorting arrays in place during React renders. The safe way to do this is by making a copy of the array (`array.concat().sort()`) but that means the logic here will need to be tweaked a bit. I think this calls for a helper function:

```ts
getOrderedBylineContributors(contributors: Contributor[], ordering: BylineOption) => Contributor[]
```
where `Contributor` is (mumble mumble) a `string | Attribution`.